### ISSUE TITLE
Error exporting audio from PUA?

### ISSUE THREAD

**Issue by annewootton** [popuparchive/pop-up-archive#1357]
From a user: "when I try to export audio, I get this error message (Chrome for Mac / El Capitan)":
```
<Error slick-uniqueid="3">
<div>
<a id="slick_uniqueid"/>
</div>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>AA15ACD4332FC702</RequestId>
<HostId>
IR/GcGO8dzGLOsxDyFs7/T1bFbSNS5fcXIi3pNVRqaFw4JdRawkhgF+UMfVrBoi7DE0WGtxSvoE=
</HostId>
</Error>
```

**Comment by baileyspace** [comment]
Can you get the item number/url?

**Comment by annewootton** [comment]
Will do.

**Comment by Saltzshaker** [comment]
@annewootton did you track down the item? I just exported an audio file and all was well

**Comment by annewootton** [comment]
Yes! It's https://www.popuparchive.com/collections/431/items/48906

**Comment by Saltzshaker** [comment]
This looks like an IA issue. The file plays in Pop Up and was copied over to s3 from IA. When I click the original_file_url in the API, it throws a 404 error. I'm guessing this is the file we're trying to grab when you click export audio

**Comment by annewootton** [comment]
Oh, that makes sense — I think it's because at some point we made our
archive.org materials stream only... though I thought that was just for
Studs Terkel. Worth investigating and if necessary we can ask archive.org
to make the rest of our (non-Studs) stuff downloadable.

**Comment by annewootton** [comment]
Update: I'm able to download https://archive.org/download/space-shuttle-challenger-disaster-archival-newsca.SW4U1E.popuparchive.org/Space_shuttle_Challenger_explodes_01_28_29_1986.mp3 when logged in with privs. In fact, I think perhaps anyone can download this file by right-clicking. 

I am checking with archive.org staff to find out if we should indeed be able to download these files when authenticated to archive.org as Pop Up Archive.

**Comment by annewootton** [comment]
Files should be downloadable. 

From archive.org staff: "You should be able to download using either wget or the ia wrapper. What method were you trying to use?"

@AndersEverett can you look into this and see if you can learn anything more about why we're getting errors when we try to download from archive.org? See above comments for sample item(s).
